From ba93068e49b89374de1286db907d403fe3b3fbfe Mon Sep 17 00:00:00 2001
From: Minecrell <minecrell@minecrell.net>
Date: Thu, 18 Feb 2021 12:17:41 +0100
Subject: [PATCH 6/6] net: qualcomm: bam-dmux: Set some additional flags

Those are also set by qmi_wwan and mhi_net, not sure about IFF_MULTICAST.
---
 drivers/net/ethernet/qualcomm/bam-dmux.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/qualcomm/bam-dmux.c b/drivers/net/ethernet/qualcomm/bam-dmux.c
index 33f4c4cf8557..749ae8d25d7f 100644
--- a/drivers/net/ethernet/qualcomm/bam-dmux.c
+++ b/drivers/net/ethernet/qualcomm/bam-dmux.c
@@ -443,6 +443,7 @@ static void bam_dmux_netdev_setup(struct net_device *dev)
 
 	dev->type = ARPHRD_RAWIP;
 	SET_NETDEV_DEVTYPE(dev, &wwan_type);
+	dev->flags = IFF_POINTOPOINT | IFF_NOARP;
 
 	dev->mtu = ETH_DATA_LEN;
 	dev->max_mtu = BAM_DMUX_MAX_DATA_SIZE;
-- 
2.30.1

